apps/studio/package.json

```diff
@@ -52,8 +49,6 @@
     "file-saver": "^2.0.5",
     "framer-motion": "^6.5.1",
     "generate-password": "^1.7.0",
-    "graphql": "^16.8.1",
-    "graphql-ws": "^5.14.1",
```

Comment:
These might still be needed for GraphiQL

Reply:
I've successfully triggered some graphl requests, what else should I test?

These might also be transitive dependencies from graphiql

Reply:
Already a transitive dep, shouldn't be a need to define it separately

Reply:
Noice!